sound/core/pcm.c

```diff
@@ -816,20 +816,12 @@ int snd_pcm_new_internal(struct snd_card *card, const char *id, int device,
 }
 EXPORT_SYMBOL(snd_pcm_new_internal);
 
-static void free_pcm_kctl(struct snd_pcm_str *pstr)
+static void free_chmap(struct snd_pcm_str *pstr)
 {
 	if (pstr->chmap_kctl) {
 		snd_ctl_remove(pstr->pcm->card, pstr->chmap_kctl);
 		pstr->chmap_kctl = NULL;
 	}
-	if (pstr->vol_kctl) {
-		snd_ctl_remove(pstr->pcm->card, pstr->vol_kctl);
-		pstr->vol_kctl = NULL;
-	}
-	if (pstr->usr_kctl) {
-		snd_ctl_remove(pstr->pcm->card, pstr->usr_kctl);
-		pstr->usr_kctl = NULL;
-	}
```

Comment:
also drop, patch adds no fixes but that 696969% placebo improvements by renamed functions and removed checks...